### ISSUE TITLE
Missing device configuration: Z-Wave.Me Tion 4S controller stick

### ISSUE THREAD

**Issue by GamePad64** [AlCalzone/node-zwave-js#3639]
### Checklist

- [X] It is not in the [configuration DB](https://devices.zwave-js.io/?search=3)

- [X] It was not [merged recently](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Apr+is%3Aclosed) or has a [pending PR](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Aopen+is%3Apr)

### Which device is missing?

Tion 4S Z-Wave control stick

### Manufacturer ID

0x0115

### Product Type

0x0100

### Product ID

0x0910

### Firmware Version

2.17

### Do you have a manual?

Manual in Russian: [ZME_TION_4s.html.pdf](https://github.com/zwave-js/node-zwave-js/files/7456982/ZME_TION_4s.html.pdf)
I can help with the translation, if needed.

Basically, the device is a control stick, that can be attached to a forced ventilation system.
Also, here is debug info for the device: https://gist.github.com/GamePad64/a72d754fe3066e8c58f36f2633554b90

**Comment by AlCalzone** [comment]
@zwave-js-bot config help wanted
